### PR TITLE
Only run docker integration on merge not on PR and timeout docker int…

### PR DESCRIPTION
**Pull request by mhagen-vmware**
To help with build process right now.

### CODE REVIEW

File: tests/integration-test2.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -x
+
+gsutil version -l
+
+dpkg -l > package.list
+pybot --removekeywords TAG:secret tests/test-cases/Group1-Docker-Commands tests/test-cases/Group2-Imagec tests/test-cases/Group3-vic-machine
```
Comment (hickeng) [inline tests/integration-test2.sh]: does this mean we're dropping imagec tests and vic-machine tests as well?

**Comment by hickeng** [comment]
LGTM

**Comment by andrewtchin** [comment]
lgtm

**Comment by mhagen-vmware** [comment]
.

**Pull request merged**